code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1437,7 +1437,7 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	default_cell_type = /obj/item/stock_parts/cell/bluespace
 
 /mob/living/silicon/robot/destroyer/init(alien = FALSE, mob/living/silicon/ai/ai_to_sync_to = null)
-	..()
```

Comment:
This parent call handles more than just laws and AI syncing, such as adding a camera and the binary channel for law stating. I would re-add those unless its 100% intentional to not have either
```dm
	aiCamera = new/obj/item/camera/siliconcam/robot_camera(src)
	make_laws()
	additional_law_channels["Binary"] = ":b "
	var/found_ai = ai_to_sync_to
	if(!found_ai)
		found_ai = select_active_ai_with_fewest_borgs()
	if(found_ai)
		lawupdate = 1
		connect_to_ai(found_ai)
	else
		lawupdate = 0

	playsound(loc, 'sound/voice/liveagain.ogg', 75, 1)
```

Reply:
Originally it was intentional to not have either.
Having slept on it though, I've decided to add them. I don't expect either will be used, and I'm not even sure the camera is a good thing, but for the sake of keeping changes minimal, they're now added.